project_generator/tools/uvision.py

```diff
@@ -69,6 +71,10 @@ class uVisionDefinitions():
             'Utilities': {
                 'Flash2': 'STLink\\ST-LINKIII-KEIL_SWO.dll',
             },
+            'OptxFile' : {
```

Comment:
I think it would be better to organize this dictionary. ``Utilities`` is for uvproj file, thus shall be under ``uvproj`` and ``OptxFile`` should be under ``uvopt``? Mixing two different files within one dictionary might lead to confusion later. I had to search for usage of OptxFile to see if it's applied to uvproj or optx :-)

What do you think?